src/go/rpk/pkg/cli/cmd/redpanda/config.go

```diff
@@ -125,7 +125,11 @@ func bootstrap(mgr config.Manager) *cobra.Command {
 				},
 			}}
 
-			conf.Redpanda.AdminApi.Address = ownIp.String()
+			conf.Redpanda.AdminApi = []config.NamedSocketAddress{{
+				SocketAddress: config.SocketAddress{
+					Address: ownIp.String(),
```

Comment:
You'll have to manually set the default port too. It wasn't necessary previously, but because here you're rewriting the whole list, the port will be 0.
See https://github.com/vectorizedio/redpanda/blob/dev/src/go/rpk/pkg/cli/cmd/redpanda/config.go#L124

Reply:
Thanks!
Added.